config.std

```diff
@@ -57,4 +57,4 @@ $(CFG): $(CFG_DEPENDENCIES) ../config.mak
 
 #		*Implicit Rules*
 .asm.obj:
-	$(NASM) -f obj $(NASMFLAGS) -DMODEL=$(SHELL_MMODEL) -DCOMPILER=$(COMPILER) $<
+	$(NASM) -f obj $(NASMFLAGS) -DMODEL=$(SHELL_MMODEL) -DCOMPILER=$(COMPILER) -o $@ $<
```

Comment:
I am not sure if $@ works for all the make varieties that can be used. As far as I know the $@ is not necessary here anymore since you commented out the -f elf above and `sed` takes care of it for GNU make?